python/ray/dashboard/modules/aggregator/publisher/async_publisher_client.py

```diff
@@ -125,3 +174,100 @@ def set_session(self, session) -> None:
         If a session is set explicitly, it will be used and managed by close().
         """
         self._session = session
+
+
+class AsyncGCSPublisherClient(PublisherClientInterface):
```

Comment:
```suggestion
class AsyncGCSTaskEventsPublisherClient(PublisherClientInterface):
```

Want to make it explicit that we only publish task events to GCS